modules/workflows/OrcaCopyToArchiveWorkflow/orca_copy_to_archive_workflow.asl.json

```diff
@@ -4,19 +4,17 @@
   "States": {
     "CopyToArchive": {
       "Parameters": {
-        "cma": {

```

Comment:
Wrong lambda

Reply:
Concur, this is the wrong lambda and should be reverted.

Reply:
reverted. I forgot the change was made only in extract_filepath for now